resolwe_bio/migrations/resolwe_indexes_3.sql

```diff
@@ -0,0 +1,50 @@
+-- Index for Django filter descriptor__general__species__icontains
+CREATE INDEX idx_collection_descriptor__general__species ON flow_collection USING gin (UPPER(descriptor #>> '{general,species}'::text[]) gin_trgm_ops );
```

Comment:
All indices have to be applied to the `flow_entity` table. Fix index names accordingly.